corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -183,8 +183,11 @@ def validate_fixture_upload(workbook):
         tag = table_def.table_id
         fields = table_def.fields
         item_attributes = table_def.item_attributes
-        data_items = list(workbook.get_data_sheet(tag))
-        for sort_key, data_item in enumerate(data_items):
+        try:
+            data_item = iter(workbook.get_data_sheet(tag)).next()
```

Comment:
can you explain why this no longer needs to loop through all of the `data_items`?